app/mailers/patient_mailer.rb

```diff
@@ -225,60 +261,4 @@ def closed_email(patient)
       format.html { render layout: 'main_mailer' }
     end
   end
-
```

Comment:
So I actually think these should still be pulled out into separate methods. There are lines and snippets repeated multiples times and pulling them out is the best way to ensure consistency in the comments and avoid code duplication. Especially the lines of code repeated for updating the history after a successful report reminder.

Reply:
Ok let me take a look